src/content/runtime-apis/scheduled-event.md

```diff
@@ -0,0 +1,42 @@
+# ScheduledEvent
+
+## Background
+
+The event type for Scheduled requests to a Worker. The `Object` passed through as `event` in 
+[`addEventListener()`](/runtime-apis/add-event-listener).
+
+## Context
+
+```js
+addEventListener("scheduled", event => {
+  event.waitUntil(handleScheduled(event))
+})
+```
+
+### Properties
+
+<Definitions>
+
+- `event.type` <Type>string</Type>
```

Comment:
is this `ScheduleEvent` in this situation or will it ever be something different?

Reply:
It should be `scheduled`, like how `FetchEvent` returns `fetch`.